operations_test.go

```diff
@@ -348,6 +348,58 @@ func TestGt(t *testing.T) {
 
 }
 
+func GetF64 (n *Node) float64 {
```

Comment:
Is there a simpler way to do this? I need to get the value of the node in order to compare it with assert.Greater/assert.Less.

Reply:
https://github.com/gorgonia/gorgonia/blob/master/testsetup_test.go#L53